### ISSUE TITLE
[Suggestion] - Differentiate /home and /homes to make them more useful and the home system faster and better

### ISSUE THREAD

**Issue by ironman92710** [EssentialsX/Essentials#3957]
### Feature request

**Feature description:**  
Make the /home command teleport you to your first or some 'default' home if you don't specify a home when you have multiple homes (and /home "name" is what you use to go to a specific home like how it works now). Make /homes just list your home's names like /home does now if you don't specify a home with multiple homes

**How the feature is useful:**  
Most players have a main base or bed they are using /home for 90% of the time. Even if they have multiple homes they set them in places they want to have an anchor at but not visit as often. This makes the home system significantly faster and more fluid and is done a lot as an optimization in a lot of home plugins and servers. With this /home is faster for ~90% of its uses (which would be teleporting to that first base) with essentially zero downsides. It also makes /homes have its own use which is less confusing than it just being a weird clone of /home with a different name and perm.

**Comment by seema84** [comment]
I would also like that (I was just about to make the same suggestion)
a first default home (maybe mark as default) I also find very intuitive

i would also appreciate it if the exact coordiants incl. world are displayed under /homes

**Comment by molor** [comment]
You are not the first to come up with this enhancement. We are waiting for many years for this to be added, and still all that remains is to keep waiting..

https://github.com/EssentialsX/Essentials/pull/881
https://github.com/EssentialsX/Essentials/issues/893

**Comment by seema84** [comment]
Is there no way to change this before the 5.0.0 milestone? 
It is enough if you are always teleported to the default home (i.e. the first or another home set as default) with /home.

**Comment by mdcfe** [comment]
While I definitely want to do this at some point, there isn't really a sensible way of doing this in 2.x without confusing existing users. I'd rather hold this back until we're ready to make major changes to the structure of many commands at once.

**Comment by ironman92710** [comment]
I don't think this will be nearly as confusing as you think it might be. The reason for this is it is the way it's already done on a great number of servers and I've had many conversations on discord about the current system being confusing (multiple commands for the same thing etc.). I think it would be fine if this change came in a major update like has been mentioned before but I think it would be really nice now as well. It will probably be the case that many people (like me) won't be confused when they were otherwise confused by essentialsX when they first installed it and people who keep up with the updates may be momentarily confused and a /help command explaining it would solve that pretty promptly. It is a more useful improved system the way I described and I think that improvement will likely lead to a similar level of confusion as there is now but with a better system.

**Comment by mdcfe** [comment]
Essentials and EssentialsX have had this exact behaviour for *over a decade* (!) - the proposed change would be disruptive to anyone who has used Essentials or EssentialsX over the past 10 years. 2.x isn't the place to start breaking people's well-established expectations, especially ones that go back *this* far. The next major version of EssentialsX may provide the option to change this, hence the 5.0.0 milestone, but as it stands I'm not willing to introduce a massive breaking UX change into 2.x.

**Comment by ironman92710** [comment]
This explanation I understand a lot more and it's what I was told before. If it's a big UX change and influences a lot of other things outside of /home (which it sounds like it does if it's at the core of a lot of essentialsX's functionality) then reconfiguring other things COULD cause a lot of confusion. Another thing is 5.0.0 will be adopted or not basically simultaneously to what will either become the last few updates or continued support for 2.x. It's not like this change will be any more or less jarring for Essentials players in 5 as it would be in 2.x. It's just with 5 people could still choose to have 2.x as it's likely being developed unlike making it a 2.x.x change which is pretty commital as people would have to stop installing future updates to 2 in order to get the /home behavior they prefer. Another option is to create the new system and let people choose in their EssentialsX. Make the proposed /home behavior and then have it only be enable-able in the config or etc. That way you could improve /home and improve people's onboarding to 5.0's system by organically allowing people to choose the faster system in 2. Again this is only meant to address an adoption problem among the users and not any underlying systems that might break with this change. If it's arduous and difficult and is going to hamper 5.0's production then by all means focus on that. I'd imagine basically making it a back door macro that implements the commands in the configuration described would be pretty simple to implement on the surface without it affecting the underlying structure of EssentialsX.
